src/AssetsDetectionController.ts

```diff
@@ -97,18 +97,21 @@ export class AssetsDetectionController extends BaseController<AssetsDetectionCon
 			tokens: []
 		};
 		this.initialize();
+		this.poll();
 	}
 
 	/**
-	 * Sets a new polling interval
+	 * starts a new polling interval
 	 *
 	 * @param interval - Polling interval used to auto detect assets
 	 */
-	set interval(interval: number) {
-		this.handle && clearInterval(this.handle);
-		this.handle = setInterval(() => {
-			safelyExecute(() => this.detectAssets());
-		}, interval);
+	async poll(interval?: number) {
+		this.config.interval = interval || this.config.interval;
```

Comment:
is this working?  I believe you should call `this.configure()` to change the controller config (same applies to the other controller `poll` methods)